docs/src/main/sphinx/release/release-449.md

```diff
@@ -0,0 +1,78 @@
+# Release 449 (31 May 2024)
+
+## General
+
+* Add an event listener which exposes collected events to an HTTP endpoint. ({issue}`22158`)
+* Fix rare query failure or incorrect results for array types when the data is
+  dictionary encoded. ({issue}`21911`)
```

Comment:
How rare is the "rare"? What does it mean from user perspective?
Asking for guidance when formulating future proposed release notes entries.